.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,7 +1,54 @@
 <component name="InspectionProjectProfileManager">
```

Comment:
Could you undo the changes you made into these files? I'm going to look to see what I can do about these files (either remove them, or make them in such a way that they shouldn't be modified).